addons/addon-base-raas-ui/packages/base-raas-ui/src/helpers/api.js

```diff
@@ -338,6 +338,10 @@ function updateRegisteredAccount(accountId, data) {
   });
 }
 
+function getEgressStore(environmentId) {
+  return httpApiGet(`api/data-egress/${environmentId}`);
+}
+
```

Comment:
This API doesn't exist yet I am guessing and there will be a separate PR on this?